playwright-e2e/dsm/component/navigation/navigation.ts

```diff
@@ -64,6 +64,50 @@ export class Navigation {
     await this.selectFrom(MainMenuEnum.MISCELLANEOUS, miscName);
   }
 
+  public async getDisplayedMainMenu(): Promise<MainMenuEnum[]> {
+    let menuOptions: Locator[] = [];
+    await expect(async () => {
+      menuOptions = await this.page.locator(`//app-navigation//ul[1]//li[contains(@class, 'dropdown')]//b`).all();
+      const amountOfMenuOptions = menuOptions.length;
+      console.log(`Amount of main menu options: ${amountOfMenuOptions}`);
+      expect(amountOfMenuOptions).toBeGreaterThanOrEqual(1);
+    }).toPass({
+      intervals: [5_000],
+      timeout: 30_000
+    });
+
+    const displayedMenuOptions: MainMenuEnum[] = [];
+    for (const option of menuOptions) {
+      const optionName = ((await option.innerText()).trim()) as MainMenuEnum;
+      console.log(`Main menu option: ${optionName}`);
+      displayedMenuOptions.push(optionName);
+    }
+    return displayedMenuOptions;
+  }
+
+  public async getDisplayedSamplesMenuOptions(): Promise<SamplesNavEnum[]> {
+    let menuOptions: Locator[] = [];
+    await expect(async () => {
+      menuOptions = await this.page.
+      locator(`//app-navigation//ul[1]//li[contains(@class, 'dropdown')]//a[contains(.,'Samples')]/following-sibling::ul//li`).
+      all();
+      const amountOfMenuOptions = menuOptions.length;
```

Comment:
Maybe `numMenuOptions` or `countOfMenuOptions` would probably be more clear, although that might just be me.  With "amount", my mind drifted to "what are the units for this amount?"